frontend/android/app/build.gradle

```diff
@@ -123,12 +133,19 @@ android {
         }
     }
 
+    def buildConfigName = determineBuildConfigName()
+    def buildConfig = createBuildConfig(buildConfigName)
+
     defaultConfig {
         minSdkVersion 21
         targetSdkVersion 33
         multiDexEnabled true
         versionCode flutterVersionCode.toInteger()
         versionName flutterVersionName
+        if (project.hasProperty('VERSION_CODE')) {
```

Comment:
why can this code be optional? Shouldn't we always have a version code?

Reply:
Only for fastlane builds we set the version code as a project property. When you build if locally the flutterVersionCode is used. For builds with fastlane/circleci we us the version the toolbelt calcultaes and overwrite the flutterversion